composer.json

```diff
@@ -31,7 +31,7 @@
         "symfony/options-resolver": "^2.3 || ^3.0"
     },
     "require-dev": {
-        "friendsofsymfony/rest-bundle": "^1.1",
+        "friendsofsymfony/rest-bundle": "<1.1",
```

Comment:
it is wrong. I confused the confict and require-dev section. Already made a fix...if i'm not wrong it should be automatically added here, right?